.github/reviewers.yml

```diff
@@ -0,0 +1,74 @@
+reviewers:
+  # The default reviewers
+  defaults:
+    - repository-owners # group
+
+  # Reviewer groups each of which has a list of GitHub usernames
+  groups:
+    repository-owners:
+      - wbreza
+      - fnocera
+    agile:
+      - meulta
+      - omri374
+    automated-testing:
+      - alkofahi
+      - omri374
+    code-reviews:
+      - katyamust
+      - TessFerrandez
+    ci-cd:
+      - brockneedscoffee
+    design: 
+      - zmmille2
+    dev-experience:
+      - olasowemimo
+    div-inclusion:
+      - madossan01
+    documentation: 
```

Comment:
Feel free to add me to documentation and machine-learning as well, and tompaana to documentation and source-code so we have at least two, omri374 could probably go on machine-learning too

Reply:
Feel free to add me to privacy as well. Thanks! Great addition

Reply:
Added you!